### PR TITLE
Add docker-compose

### PR DESCRIPTION
**Pull request by julianaklulo**
#### What
Add Dockerfile and docker-compose to lm-sim.

#### Why
We need a repeatable way to run the project.

`Task`: https://app.clickup.com/t/18022949/LM-183

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+
+ENV MODULE_NAME=license_manager_simulator.main
+
+RUN apt update && apt install -y curl
```
Comment (heitorPB) [inline Dockerfile:5]:
```suggestion
RUN apt-get update && apt-get install -y curl
```

`apt` should not be used in scripts, on some situations it requests actions from the user with non-scriptable actions.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_HOME=/opt/poetry python && \
+    cd /usr/local/bin && \
+    ln -s /opt/poetry/bin/poetry && \
```
Comment (heitorPB) [inline Dockerfile:9]: Why not this:

```suggestion
RUN pip install poetry && \
```

**Review by heitorPB — CHANGES_REQUESTED** [review]
Looks great! Just missing a `.dockerignore` with at least `.git` in it.

File: Dockerfile (hunk 2 above)
Comment (jamesbeedy) [reply Dockerfile:9]: Do we even need poetry here?

Comment (heitorPB) [reply Dockerfile:9]: I think it is OK to use Poetry to install the dependencies. This is for local development only, not for a production system.

File: Dockerfile (hunk 1 above)
Comment (matheushent) [reply Dockerfile:5]: I guess adding `ARG DEBIAN_FRONTEND=noninteractive` in the beginning of the file can overcome these situations, doesn't?

File: Dockerfile (hunk 2 above)
Comment (julianaklulo) [reply Dockerfile]: I used the LM Dockerfile as a base for this one and we're installing poetry that way there as well. I tested installing it with pip and it worked, so I changed it in the latest commit.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+version: "3.9"
+services:
+
+  postgres-back:
+    image: postgres
```
[the remark this answers is not in the document]
Comment (julianaklulo) [reply docker-compose.yaml]: Sure, I changed it in the latest commit.

**Comment by julianaklulo** [comment]
> Looks great! Just missing a `.dockerignore` with at least `.git` in it.

@heitorPB I added .dockerignore, can you review again please?

**Review by heitorPB — APPROVED** [review]
Nice work!

**Pull request merged**